MessageViewController/MessageViewController.swift

```diff
@@ -86,6 +86,7 @@ open class MessageViewController: UIViewController, MessageAutocompleteControlle
     internal var scrollView: UIScrollView?
     internal var keyboardHeight: CGFloat = 0
     internal var isMessageViewHidden = false
+    internal var scrollsToBottomOnKeyboardReveal = true
```

Comment:
Can this default be `false` since it'd change behavior w/out warning to anyone that upgrades?